app/services/migration/migrator.rb

```diff
@@ -5,8 +5,15 @@ class MigrationAlreadyRanError < RuntimeError; end
     class MigrationNotPreparedError < RuntimeError; end
     class MigrationAlreadyPreparedError < RuntimeError; end
 
+    MIGRATORS = [
+      Migration::Migrators::LeadProvider,
+      Migration::Migrators::Cohort,
+      Migration::Migrators::Statement,
+      Migration::Migrators::User,
+    ].freeze
+
     class << self
-      MODELS_TO_BE_MIGRATED = %i[lead_provider cohort statement].freeze
+      MODELS_TO_BE_MIGRATED = %i[lead_provider cohort statement user].freeze
```

Comment:
I think it would make sense to encapsulate this/the prep logic in the individual migrators, that way we only need to maintain the array of `MIGRATORS` here and we can do something like:

```
def prepare_for_migration
  raise MigrationAlreadyPreparedError, "The migration has already been prepared" if DataMigration.exists?
  MIGRATORS.each(:prepare!)
end
```

And in the migrator:

```
def prepare!
  DataMigration.create!(model: :foo)
end
```

That feels more extensible as well in case we need to do more prep for a particular migrator?